src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_LENGTH = 10;
+
+    private T[] elementData = (T[]) new Object[INITIAL_LENGTH];
+    private int size = 0;
```

Comment:
let’s initialise non-static fields in the constructor

Reply:
done